README.md

```diff
@@ -1 +1,3 @@
-# calculator
\ No newline at end of file
+# calculator
+
+This program lets you calculate simple mathematics
```

Comment:
instead of mathematics, please use mathematical operations

Reply:
Addressed